libreconv.gemspec

```
@@ -23,8 +23,9 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_development_dependency 'bundler', '>= 0'
+  spec.add_runtime_dependency 'mixlib-shellout', '~> 3.3', '>= 3.3.4'
```

It's best not to be mixed otherwise it might get confusing like the example here https://github.com/ScreenLoop/hubspot-rb/blob/master/hubspot-api-client.gemspec
```suggestion
  spec.add_runtime_dependency 'mixlib-shellout', '~> 3.3', '>= 3.3.4'

  spec.add_development_dependency 'bundler', '>= 0'
```